src/app/components/value-indicator/value-indicator.component.scss

```diff
@@ -0,0 +1,42 @@
+:host {
+    display: block;
+    width: 100%;
+    height: 12px;
+
+    .indicator-container {
+        position: relative;
+        width: 100%;
+        height: 100%;
+        .indicator {
+            width: 12px;
+            height: 12px;
+            border-radius: 50%;
+            border: 2px solid black;
+            background-color: #ddd;
+            position: absolute;
+            top: 0;
+            left: 0;
+            transition: transform 0.3s ease;
+        }
+        .slider {
+            display: block;
+            width: 100%;
+            height: 1px;
```

Comment:
In 27" full HD monitor connected to macbook pro the lines are not visible - probably too thin. On the macbook monitor it works fine. This is probably some crazy black magic which macos does with pixels on non-retina monitors.

I can debug this in follow up issue.

Reply:
![image](https://github.com/user-attachments/assets/00f0a7f2-fe0c-4735-95b1-98eca7584f9e)

Reply:
Lol, firefox screenshot just added the lines :D but it is not visible for user. With system screenshot it is... what you see is what you get:
![Screenshot 2025-05-08 at 23 26 24](https://github.com/user-attachments/assets/31ffe0a2-a93f-4ff9-abfe-739f18fc2250)